nuget/Autofac.Extras.MvvmCross.nuspec

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This information should be taken from the project.json file as part of the build. Why is this .nuspec file required?